website/content/docs/connect/config-entries/service-intentions.mdx

```diff
@@ -86,20 +86,19 @@ The following outline shows how to format the service intentions configuration e
     - [`permissions`](#spec-sources-permissions): list | no default
       - [`action`](#spec-sources-permissions-action): string | no default  | required
       - [`http`](#spec-sources-permissions-http): map | required
-        - [`pathExact`](#spec-sources-permissions-http): string | no default 
-        - [`pathPrefix`](#spec-sources-permissions-http): string | no default 
-        - [`pathRegex`](#spec-sources-permissions-http): string | no default 
-        - [`methods`](#spec-sources-permissions-http): list | no default 
-          - [`header`](#spec-sources-permissions-http-header): list of maps |no default 
+        - [`pathExact`](#spec-sources-permissions-http): string | no default
+        - [`pathPrefix`](#spec-sources-permissions-http): string | no default
+        - [`pathRegex`](#spec-sources-permissions-http): string | no default
+        - [`methods`](#spec-sources-permissions-http): list | no default
+          - [`header`](#spec-sources-permissions-http-header): list of maps |no default
             - [`name`](#spec-sources-permissions-http-header): string | required
-            - [`present`](#spec-sources-permissions-http-header): boolean | `false` 
+            - [`present`](#spec-sources-permissions-http-header): boolean | `false`
             - [`exact`](#spec-sources-permissions-http-header): string | no default
             - [`prefix`](#spec-sources-permissions-http-header): string | no default
             - [`suffix`](#spec-sources-permissions-http-header): string | no default
             - [`regex`](#spec-sources-permissions-http-header): string | no default
-            - [`invert`](#spec-sources-permissions-http-header): boolean | `false` 
-    - [`type`](#spec-sources-type): string | `consul` 
```

Comment:
`type` isn't actually required when deploying on k8s, I got errors saying this field is unknown:
```
Error from server (BadRequest): error when creating "./consul/jwt-intention.yaml": ServiceIntentions in version "v1alpha1" cannot be handled as a ServiceIntentions: strict decoding error: unknown field "type"
```